### DISCUSSION TITLE
Docker - authRefresh fails

### DISCUSSION THREAD

**Discussion by peterhijma** [pocketbase/pocketbase#2995] [Q&A]
Probably this question doesn't belong here. Feel free to say so and I'll close this immediately.

I'm using Nuxt 3, including the SSR example in the JS SDK readme.

Running it in the terminal with `npm run dev` => no problems.

Now I've set up Docker (compose) and when I refresh the page, `await pb.collection("users").authRefresh()` fails, so that the user is logged out on a page refresh.

I'm not sure what is happening and trying stuff for too long 😢 I've checked the cookie before the method is called and it shows the correct value.

EDIT: Okay, so the Nuxt 3 container is talking to the Pocketbase container, that's where it's probably going wrong 🤔

**Comment by ganigeorgiev** [comment]
Are you sure that you are using the correct PocketBase URL when initializing the JS SDK?
Note that `localhost`/`127.0.0.1` inside one container will differ from the localhost in another (_you can use the targeted service_name:port to access the targeted localhost from the other containers_).
